tox.ini

```
@@ -4,7 +4,8 @@ envlist =
     py38-django{32,40,41,42}-{postgresql,sqlite}
     py39-django{32,40,41,42}-{postgresql,sqlite}
     py310-django{32,40,41,42}-{postgresql,sqlite}
-    py311-django{41,42}-{postgresql,sqlite}
+    py311-django{41}-{postgresql,sqlite}
+    py311-django{42}-{postgresql-psycopg3}
```

We still want to test sqlite on Python 3.11 and Django 4.2.

```suggestion
    py311-django{42}-{postgresql-psycopg3,sqlite}
```

---

Oh my, true, that one should be there as well. It is not such a big overhead IMHO, but important to have. So, do you mind adding it as well? 🙏

---

This is added now.